libs/barista-components/sunburst/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "types": ["node", "jest"]
+  },
+  "include": ["**/*.ts", "../../../apps/dev/src/sunburst/sunburst-test-data.ts"]
```

Comment:
Please remove the include of the testData in this tsconfig. I don't think you will need it in the component itself.